python/tvm/relay/op/contrib/tensorrt.py

```diff
@@ -709,7 +709,9 @@ def strided_slice_checker(
     if any([x is not None and x <= 0 for x in attrs.strides]):
         logger.info(f"{op_name}: stride must be positive")
         return False
-    for i in range(0, len(args[0].checked_type.shape)):
+    if len(attrs.axes) > len(args[0].checked_type.shape):
+        logger.info(f"{op_name}: number of indexed axes is higher than input tensor rank")
```

Comment:
No need for these changes, remove L712 and L713.

Reply:
done